### ISSUE TITLE
Password reset page not found

### ISSUE THREAD

**Issue by impactmass** [reactioncommerce/reaction#4911]
## Prerequisites
* [x] Are you running the latest version?
* [x] Are you able to consistently reproduce the issue?
* [x] Did you search the issue queue for existing issue? [Search issues](https://github.com/reactioncommerce/reaction/issues)

## Issue Description
Clicking on link in email sent when password reset is requested results in a 404.

## Steps to Reproduce
1. Create an account, then logout.
2. In the login dropdown, do a password reset
3. Follow link in email

## Possible Solution
By a quick check, this seem to have started after rc-6 release (needs better confirmation)

### Versions
Node: 10.13.0
NPM: 6.4.1
Meteor Node: 8.11.4
Meteor NPM: 6.4.1
Reaction CLI: 0.29.0
Reaction: 2.0.0-rc.7
Reaction branch: develop

**Comment by xtebs** [comment]
Same here. All links from email (or tokens attached) appear to be broken.
- Reset password  (blank screen - broken form)
- Invite Shop Manager #4892  
- Invite Marketplace shop owner  (blank screen - broken form)
For my team, this is currently a project killer.

**Comment by zsimoes** [comment]
> Same here. All links from email (or tokens attached) appear to be broken.
> 
>     * Reset password  (blank screen - broken form)
> 
>     * Invite Shop Manager #4892
> 
>     * Invite Marketplace shop owner  (blank screen - broken form)
>       For my team, this is currently a project killer.

Same symptoms here.

**Comment by impactmass** [comment]
Thanks @xtebs @zsimoes. We should have a fix ASAP

**Comment by impactmass** [comment]
I opened the PR linked above for the password reset. I discovered that the other issue mentioned (4892) is caused by a different reason. That will be dealt with in another PR

**Comment by NileThayne** [comment]
When I hit these problems, I found that some of them could be resolved by going to the admin dashboard, choosing Accounts, click on Groups, selecting the "Guest" group, and enabling the "Password-Reset" privilege for that group. This allows the person who is not logged in to see the password reset form, and reset their password.

**Comment by impactmass** [comment]
@NileThayne yes, you just described the manual way of granting the roles. Guest roles like this can be made available by default without expecting the admin to use the UI. That is what is done in #4917. The fix is already merged into the develop branch and will go out with the next release.

**Comment by NileThayne** [comment]
Awesome! I just wanted to clarify for anyone who might be stuck on this issue while waiting for the next release. Thanks for your work!
